### ISSUE TITLE
Assumes go-1.2

### ISSUE THREAD

**Issue by spikebike** [nictuku/wherez#1]
Pretty sure your sha256.sum256 assumes go-1.2.  The default go on most distros is an older go.  I'd either use the older calls or at least mention requiring 1.2 in the README.

With ubutu 13.10 (pretty recent OS), I get:

```
bill@left:~/tmp/dht/wherez$ go version
go version go1.1.2 linux/amd64

bill@left:~/tmp/dht/wherez$ go build
# _/home/bill/tmp/dht/wherez
./wherez.go:105: undefined: sha256.Sum256
```

**Comment by nictuku** [comment]
Want to send a pull request for either making it compatible with 1.1?

I'd prefer the former but don't have immediate access to 1.1.

I guess the old way was to do:

h := sha256.New().Sum()

That produces a []byte instead of an array, but it doesn't matter in this case. I can't easily test it though.

**Comment by spikebike** [comment]
I did make the 1.1 compatible changes and created a pull request.

**Comment by nictuku** [comment]
Fixed by pull #2.
